cdap-ui/app/cdap/components/DataPrep/ColumnActionsDropdown/ColumnActionsDropdown.scss

```diff
@@ -34,10 +34,17 @@
   }
 
   .action-item {
-    padding: 10px 15px;
+    padding: 10px 0px;
+    margin: 0 15px;
     cursor: pointer;
     position: relative;
 
+    &.filter-directive,
+    &.keep-column-directive,
+    &.fill-null-or-empty-directive {
+      border-bottom: 1px solid #aaaaaa;
+    }
```

Comment:
don't do this border for now. Eventually this should just be a separator item in the dropdown list array. It's easier to manage that way